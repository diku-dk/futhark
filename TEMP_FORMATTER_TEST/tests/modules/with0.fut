-- Does 'with' work?
-- ==
-- input { 2 } output { 42 }

module type constant = { type t val x: t }

module intconstant: (constant with t = i32) = {
  type t = i32
  def x = 40
}

def main(y: i32) = intconstant.x + y
