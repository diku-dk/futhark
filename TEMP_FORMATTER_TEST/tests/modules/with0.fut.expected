1bd7187cf1289819979c8c965480a6f4
